tests/ztest/error_hook/src/main.c

```diff
@@ -76,14 +76,32 @@ static void trigger_fault_access(void)
 	printk("b is %d\n", b);
 }
 
-static void trigger_fault_divide_zero(void)
+
+/*
+ * Make the divide instruction not being optimized because it will be
```

Comment:
"Do not optimize the divide instruction..."

Reply:
Hi @nashif ,  got it fixed, thank you for correcting this.